exampleSite/content/dev/contact/contact-recaptcha.md

```diff
@@ -0,0 +1,51 @@
++++
+fragment = "contact"
+#disabled = true
+date = "2017-09-10"
+weight = 120
+background = "light"
+form_name = "defaultContact"
+
+title = "Contact fragment with ReCaptcha support"
+subtitle  = "*not working on demo page*"
+
+# PostURL can be used with backends such as mailout from caddy
+post_url = "https://example.com/mailout" #default: formspree.io
+email = "mail@example.com"
+button_text = "Send Button" # defaults to theme default
+#netlify = false
+
+# Optional google captcha
```

Comment:
Add a small note to the comment documenting, that the config won't be used, if netlify is enabled.